lib/script_base.rb

```diff
@@ -56,7 +56,7 @@ def run
     stderr.puts "*Task*: `#{result.subtask}`"
     stderr.puts "*UUIDs*: #{result.uuids.map { |uuid| "`#{uuid}`" }.join(', ')}"
     if result.messages.present?
-      stderr.puts "*Messages*:\n#{result.messages.map { |message| "    * #{message}" }.join("\n")}"
+      stderr.puts "*Messages*:\n#{result.messages.map { |message| "    • #{message}" }.join("\n")}"
```

Comment:
I would remove the indentation, I think that's why Slack isn't parsing
```suggestion
      stderr.puts "*Messages*:\n#{result.messages.map { |message| "* #{message}" }.join("\n")}"
```

Reply:
or was this meant for a different PR

Reply:
Yes, sorry - I was experimenting and did not intend this to be included with this PR. 🙂